src/gtfs/file/StopTime.ts

```diff
@@ -12,7 +12,25 @@ export interface StopTime {
   drop_off_type: 0 | 1 | 2 | 3;
   shape_dist_traveled: null;
   timepoint: 0 | 1;
-  activity: string;
+  activity: Activity;
 }
 
-export type Platform = string;
\ No newline at end of file
+export type Platform = string;
+
+export type Activity = string | null;
```

Comment:
shouldn't it be `ActivityPattern | null` ?

Reply:
I've tried but
`Enum ActivityPattern with same name already exists in the scope`